cloudwatch_monitoring_role/main.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_iam_policy_document" "cloudwatch_monitoring_policy" {
+  statement {
+    sid     = "1"
```

Comment:
Can you give the statement id a descriptive name?

Reply:
from the name I thought it has to be an integer, although it's not very clear in the documentation https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html#sid

Reply:
Never had problems using descriptive names, e.g.: https://github.com/skyscrapers/tablebooker/blob/master/terraform/stacks/tablebooker-cron/iam.tf#L22